packages/flutter_tools/test/src/context.dart

```diff
@@ -35,7 +35,7 @@ typedef dynamic Generator();
 void testUsingContext(String description, dynamic testMethod(), {
   Timeout timeout,
   Map<Type, Generator> overrides: const <Type, Generator>{},
-  bool skip: false,
+  bool skip,
```

Comment:
I might have added a comment here saying that this should default to false and referencing the bug. :)  Otherwise the next person who comes along may not understand.